Applications/SimpleUI/static/simpleui.js

```diff
@@ -0,0 +1,103 @@
+
+        // Build URL for Telemetry db query, then retrieve data
+        var url = "http://localhost:8001/?limit=1&callsign=" +
+            callsign + // jshint ignore:line
+            "&nodeid=" + 
+            nodeid; // jshint ignore:line
+            $.getJSON(url,function(data){
+                try{
+                    // Convert EPOCH from telemetry into date object
+                    var epoch = data[0].EPOCH;
+                    var d = new Date(0);
+                    d.setUTCSeconds(epoch);
+                    $(".EPOCH").text(d);
+
+                    // Populate the GPS section
+                    $(".CALLSIGN").text(data[0].SOURCECALLSIGN);
+                    $(".NODEID").text(data[0].SOURCEID);
+                    $(".GPSLATITUDE").text(data[0].GPSLATITUDE);
+                    $(".GPSLATITUDEDIR").text(data[0].GPSLATITUDEDIR);
+                    $(".GPSLONGITUDE").text(data[0].GPSLONGITUDE);
+                    $(".GPSLONGITUDEDIR").text(data[0].GPSLONGITUDEDIR);
+                    $(".ALTITUDE").text(data[0].GPSALTITUDE);
+                    $(".SPEED").text(Math.round(data[0].GPSSPEED*0.514444*100)/100);
+                    $(".GPSFIX").text(data[0].GPSFIX);
+
+                    // Populate the ADC section
+                    $(".BOARDTEMP").text(data[0].BOARDTEMP);
+                    $(".ADC0").text(Math.round(data[0].ADC0*1000*(2.41/4096))/1000);
+                    $(".ADC1").text(Math.round(data[0].ADC1*1000*(2.41/4096))/1000);
+                    $(".ADC2").text(Math.round(data[0].ADC2*1000*(2.41/4096))/1000);
+                    $(".ADC3").text(Math.round(data[0].ADC3*1000*(2.41/4096))/1000);
+                    $(".ADC4").text(Math.round(data[0].ADC4*1000*(2.41/4096))/1000);
+                    $(".ADC5").text(Math.round(data[0].ADC5*1000*(2.41/4096))/1000);
+
+                    // Populate the High Altitude Balloon section
+                    $(".HABTIMER").text(data[0].HABTIMER);
+                    $(".HABTIMERSTATE").text(data[0].HABTIMERSTATE);
+                    $(".HABTRIGGERTIME").text(data[0].HABTRIGGERTIME);
+                    $(".HABCUTDOWNSTATE").text(data[0].HABCUTDOWNSTATE);
+
+                    // Convert GPIO data into an array of GPIO values
+                    var gpio = [];
+                    gpio = ("000000000" + data[0].GPIOSTATE.toString(2)).substr(-8).split("");
+
+                    // Convert IOSTATE data into array of binary values
+                    var iostate = [];
+                    iostate = ("000000000" + data[0].IOSTATE.toString(2)).substr(-8).split("");
+                    iostate.reverse();
+
+                    // Convert RFSTATE data to an array of values
+                    var rfstate = [];
+                    rfstate = ("000000000" + data[0].RFSTATE.toString(2)).substr(-8).split("");
+                    rfstate.reverse();
+
+                    // Populate GPIO data in IO section
+                    $(".GPIO0").text(gpio[0]);
```

Comment:
Using the [new firmware](https://github.com/FaradayRF/Faraday-Firmware/issues/72#issuecomment-295424504) I can clearly see the following when I print out the GPIO state in javascript from SimpleUI:

GPIO7 HIGH
![image](https://cloud.githubusercontent.com/assets/331540/25217662/99b6d79a-255c-11e7-8e5f-08f8127df73a.png)

GPIO0 HIGH
![image](https://cloud.githubusercontent.com/assets/331540/25217749/ea62e85a-255c-11e7-9014-0e6c72630d34.png)

GPIO1 HIGH
![image](https://cloud.githubusercontent.com/assets/331540/25217944/b7ce74bc-255d-11e7-8135-607e71e367dd.png)


Given the documentation showing the following GPIO State packet I believe this is simple reversed in the Javascript array:
![image](https://cloud.githubusercontent.com/assets/331540/25217821/2dcbf8e8-255d-11e7-9707-d6eac265b1f2.png)

However... Measuring with a multimeter to confirm GPIO output state...

# Commanding GPIOs
It appears commanding is messed up. Here's what happens when I send commands to Faraday GPIO

GPIO0: D2
GPIO1: D1
GPIO2: D0
GPIO3: D3
GPIO4: D4
GPIO5: D5
GPIO6: D6
GPIO7: D7
GPIO8: D8

_Clearly Commanding GPIO0 through GPIO2 is incorrect_

# GPIO Telemetry
D0: GPIO7
D1: GPIO6
D2: GPIO5
D3: GPIO4
D4: GPIO3
D5: GPIO2
D6: GPIO1
D7: GPIO7
D8: GPIO8

_Clearly GPIO0 never turns on and D0-D6 telemetry is reversed on SimpleUI_